packages/ui/internal/validators/tests/validators-specs.js

```diff
@@ -0,0 +1,43 @@
+import {warn, validateRange, validateStepped} from '../validators';
+
+describe('validators', () => {
```

Comment:
please add some dedicated tests for validateRangeOnce and validateSteppedOnce. Codecov shows them as covered, but probably from tests from other components